easybuild/easyconfigs/m/mcqd/mcqd-1.0.0-GCCcore-13.2.0.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'CmdCp'
+
+name = 'mcqd'
+version = '1.0.0'
+
+homepage = 'https://gitlab.com/janezkonc/mcqd'
+description = """MaxCliqueDyn is a fast exact algorithm for finding a maximum clique in an undirected graph."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+
+source_urls = ['https://gitlab.com/janezkonc/mcqd/-/archive/v%(version)s/']
+sources = ['%(name)s-v%(version)s.tar.gz']
+checksums = ['37ff68ff88e047c929990d4c12ce474753f6f9c49324661a3aa1cfe77c26ad9d']
+
+builddependencies = [
+    ('binutils', '2.40'),
+]
+
+cmds_map = [('.*', 'echo here && pwd && ls -lh && $CXX $CXXFLAGS $LDFLAGS mcqd.cpp -o mcqd')]
+
+files_to_copy = [
+    (['%(name)s'], 'bin'),
```

Comment:
In general using `%(name)s` doesn't make a whole lot of sense, since that doesn't change across versions of the software.

So just use `mcqd` everywhere here (also in `sources`, and below)

Reply:
changed